### ISSUE TITLE
wavenet train.py loss exloded

### ISSUE THREAD

**Issue by 11721206** [Rayhane-mamah/Tacotron-2#372]
![image](https://user-images.githubusercontent.com/19368412/56135755-fed2cc80-5fc3-11e9-90a7-901731efe2dc.png)
when i try train my mandarin model using one speaker voice by python train.py python train.py --model='Tacotron-2' with default hparam.py
the tacotron-2 traing process finished ok,
but when train wavenet, the loss exploded.
I changed the wavenet wavenet_batch_size=1, the same error occurs.
is there any suggestions for my work?

**Comment by Bobb10** [comment]
@11721206. Did you already found a solution for this issue and if so, can you share this solution. I have the same problem with a custom dataset, its loss explodes after 35K of wavenet training steps.

**Comment by 11721206** [comment]
sorry.I did not find the solution.
